Godnoken/JavaScript-IV
# JavaScript 4

## What does this PR do?
JavaScript 4 I Project solutions & Stretch goals
​
## Description of tasks completed
- Implemented code from codesandbox and refactored from prototype to classes
- Implemented Lambda students, instructors and PM's through extending Person class
- Implemented a method for instructors and PM's to increase or decrease a student's grade randomly (Stretch)
- Implemented a graduate method for students (Stretch)
​
## How can this be tested?
1. Clone the project.
2. Open up the .js files in your favourite code editor.
3. Check for changes in the console after you've ran the code.
4. Alternatively, open up index.html and look inside your browsers console for changes.
​
## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/67014113-8ffe3980-f0f4-11e9-832e-3c9fa8e5af53.png)

![image](https://user-images.githubusercontent.com/17499594/67014166-a5736380-f0f4-11e9-9491-b569154c1f08.png)
